### PR TITLE
CLOUDP-128430: Clarify roles on user creation

### PR DESCRIPTION
**Pull request by fmenezes**
## Proposed changes

Clarify roles on user creation

_Jira ticket:_ CLOUDP-128430

Closes #[issue number]

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

## Further comments

```
$ atlas dbusers create --help
Create a database user for your project.

Usage:
  atlas dbusers create [flags]

Examples:
  Create an Atlas database admin user:
  $ atlas dbuser create atlasAdmin --username <username>  --projectId <projectId>

  Create a database user with read/write access to any database:
  $ atlas dbuser create readWriteAnyDatabase --username <username> --projectId <projectId>

  Create a database user with multiple roles:
  $ atlas dbuser create --username <username> --role clusterMonitor,backup --projectId <projectId>

  Create a database user with multiple scopes:
  $ atlas dbuser create --username <username> --role clusterMonitor --scope clusterName:CLUSTER,DataLakeName:DATA_LAKE --projectId <projectId>

Flags:
      --awsIAMType string    AWS IAM method by which the provided username is authenticated. Valid values are NONE, USER, or ROLE. (default "NONE")
      --deleteAfter string   Timestamp in ISO 8601 in UTC after which Atlas deletes the user.
  -h, --help                 help for create
      --ldapType string      LDAP method by which the provided username is authenticated. Valid values are NONE, USER, or GROUP. (default "NONE")
  -o, --output string        Output format. Valid values are json, json-path, go-template, or go-template-file.
  -p, --password string      User's password.
      --projectId string     Project ID to use. Overrides the settings in the configuration file or environment variable.
      --role strings         User's roles and the databases or collections on which the roles apply.
                             The roles format is roleName[@dbName[.collection]].
                             roleName can either be a builtin role or a custom role.
                             dbName and collection are optional and only needed for builtin roles.
      --scope strings        Array of clusters and Atlas Data Lakes that this user has access to.
  -u, --username string      Username for authenticating to MongoDB.
      --x509Type string      X.509 method by which the provided username is authenticated.  Valid values are NONE, MANAGED, or CUSTOMER. (default "NONE")

Global Flags:
  -P, --profile string   Profile to use from your configuration file.
```

### CODE REVIEW

File: internal/usage/usage.go [hunk 1]
```diff
@@ -15,66 +15,70 @@
...
+	TeamID                   = "Unique identification of the team."
+	Password                 = "User's password."
+	Country                  = "The ISO 3166-1 alpha two-letter country code of the user's country of residence."
+	Mobile                   = "The user's mobile or cell phone number."
+	Period                   = "Duration in ISO 8601 that specifies how far back in the past to retrieve measurements."
+	Roles                    = "User's roles and the databases or collections on which the roles apply."
+	RolesExtended            = `User's roles and the databases or collections on which the roles apply.
+The roles format is roleName[@dbName[.collection]].
+roleName can either be a builtin role or a custom role.
+dbName and collection are optional and only needed for builtin roles.`
```
Comment (gssbzn) [inline internal/usage/usage.go:81]: should this be
```suggestion
dbName and collection are optional and only allowed for builtin roles.`
```
I guess docs can decide

**Review by gssbzn — APPROVED** [review]
LGTM

File: internal/usage/usage.go (hunk 1 above)
Comment (melissamahoney-mongodb) [inline internal/usage/usage.go:81]: I'd go with:
```suggestion
The roles format is roleName[@dbName[.collection]].
roleName can either be a built-in role or a custom role.
dbName and collection are only required for built-in roles.`
```

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**